tests/unit/replicator_tests.py

```diff
@@ -210,6 +211,47 @@ def test_create_replication(self):
             ])
         )
 
+    def test_timeout_in_create_replication(self):
```

Comment:
Any particular reasoning behind using the creation of a replication for testing the timeout? Does it clean up the replicator document correctly afterwards?